reloj-checador/app/javascript/packs/layouts/navbar.js

```diff
@@ -0,0 +1,14 @@
+controller_page = (document.getElementById("controller_name").value).toString();
+switch (controller_page) {
+    case 'attendaces':
+        document.getElementById("attendaces_li").classList.add('active');
+        break;
+    case 'companies':
+        document.getElementById("companies_li").classList.add('active');
+        break;
+    case 'employees':
+        document.getElementById("employess_li").classList.add('active');
+        break;
+    default:
+        break;
+}
```

Comment:
Add a blank line.

Reply:
Code changed. New code is in comments below